File: bee1160.java
// 1160 - 
import java.io.IOException;
import java.util.Scanner;

class bee1160 {

    public static void main(String[] args) throws IOException {
        Scanner scan = new Scanner(System.in);

        int n = scan.nextInt();

        while(n > 0) {

            int pa, pb, ano = 0;
            double ga, gb;
            boolean maisDeCem = false;

            pa = scan.nextInt();
            pb = scan.nextInt();

            ga = scan.nextDouble();
                ga /= 100;
                ga += 1;
            gb = scan.nextDouble();
                gb /= 100;
                gb += 1;

            while(pa <= pb && !maisDeCem) {

                pa *= ga;
                pb *= gb;

                ano++;

                if(ano > 100) maisDeCem = true;
            }

            if(maisDeCem) System.out.println("Mais de 1 seculo.");
            else System.out.println(ano + " anos.");

            n--;
        }


        scan.close();
    }
}

File: 1-Beginner/1160/bee1160.cpp
// 1160 - Crescimento Populacional
#include <iostream>

int main(void) {

    int n;
    std::cin>>n;

    while(n > 0) {

        int pa, pb;
        double ga, gb;
        std::cin>>pa>>pb>>ga>>gb;
    
        ga /= 100;
        ga += 1;
        gb /= 100;
        gb += 1;
    
        int ano = 0;
        bool maisDeCem = false;
    
        while(pa <= pb && !maisDeCem) {
    
            pa *= ga;
            pb *= gb;
    
            ano++;
    
            if(ano > 100) maisDeCem = true;
        }
    
        if(maisDeCem) std::cout<<"Mais de 1 seculo."<<std::endl;
        else std::cout<<ano<<" anos."<<std::endl;

        n--;
    }

    return 0;
}